venvs/common.py

```diff
@@ -10,11 +10,24 @@
 import filesystems.native
 
 
-def _create_virtualenv(virtualenv, arguments, stdout, stderr):
+def get_module_path(name):
```

Comment:
Why do all this funny stuff instead of just vendoring virtualenv.py? Normally I hate vendoring, but here virtualenv is essentially unmaintained, so it doesn't move quickly on releases (if at all).

If we didn't though want to vendor virtualenv, just doing `import virtualenv; virtualenv.__file__` is a bit easier than introducing this stuff, not to mention that `imp` is deprecated (since 2.6) so it emits warnings when you use it.

Reply:
IIRC I coded that first then I glanced at virtualenv.py and didn't feel like reviewing everything outside of main or whatever they called it. This felt safer.  If you are comfortable importing, sure.

Reply:
Though deprecation seems to be 3.4.

https://docs.python.org/2/library/imp.html
https://docs.python.org/3/library/imp.html

Reply:
Unlike pip, virtualenv's Python API *is* actually public, so in theory if not for this issue itself, we'd just call `virtualenv.create_virtualenv` and not even subprocess, but yeah :/

Reply:
Looks like you're right on imp obviously, somehow misremembered my timelines, but yeah think we can just get away with either import or vendoring.

I guess import is fine and then no need to vendor.